plugins/GSdx/GSTextureCache.cpp

```diff
@@ -379,11 +379,6 @@ GSTextureCache::Target* GSTextureCache::LookupTarget(const GIFRegTEX0& TEX0, int
 			int ww = (int)(fr.left + m_renderer->GetDisplayRect().width());
```

Comment:
I have the feeling that all the "else" branch is a complete hack. Hum, code seem to be executed only for custom resolution (it would be nice to have a comment). (yes talking to myself to understand the code)